### PR TITLE
CDAP-12084 Add documentation for DynamicPartitioner write options.

### PR DESCRIPTION
**Pull request by anwar6953**
[CDAP-12084](https://issues.cask.co/browse/CDAP-12084) Add documentation for DynamicPartitioner write options.

https://builds.cask.co/browse/CDAP-DQB383-2

### CODE REVIEW

File: cdap-docs/developer-manual/source/building-blocks/datasets/partitioned-fileset.rst [hunk 1]
```diff
@@ -221,8 +221,20 @@ Then set the class of the custom partitioner as runtime arguments of the output
...
+However, a DynamicPartitioner can also be configured to allow appending to or overwriting existing partitions.
+For instance, below is an example of configuring the same DynamicPartitioner to allow appending to a partition
+if it already exists. If it does not already exist, the partition will be created::
+
+  Map<String, String> cleanRecordsArgs = new HashMap<>();
+  PartitionedFileSetArguments.setDynamicPartitioner(cleanRecordsArgs, TimeAndZipPartitioner.class,
+                                                    DynamicPartitioner.PartitionWriteOption.CREATE_OR_APPEND);
+  context.addOutput(Output.ofDataset(DataCleansing.CLEAN_RECORDS, cleanRecordsArgs));
+
+A third option available is `CREATE_OR_OVERWRITE`, which will overwrite any contents of any
```
Comment (anew) [inline cdap-docs/developer-manual/source/building-blocks/datasets/partitioned-fileset.rst]: So far, you only introduced one option. Perhaps say, "Likewise, 'CREATE_OR_OVERWRITE'  has the effect of overwriting any contents..."

**Review by anew — COMMENTED** [review]
Only one comment

File: cdap-docs/developer-manual/source/building-blocks/datasets/partitioned-fileset.rst (hunk 1 above)
Comment (anwar6953) [reply cdap-docs/developer-manual/source/building-blocks/datasets/partitioned-fileset.rst]: Updated; thanks.

**Comment by anew** [comment]
LGTM

**Pull request merged**